README.md

```diff
@@ -1,65 +1,78 @@
+| [Linux (Debian, Alpine, Rhel, Centos, Fedora, Ubuntu)](docs/detectors//linux.md) | ✔ (via [syft](https://github.com/anchore/syft)) | ❌              |
+| [Maven](docs/detectors/maven.md)                                                 | ✔                                               | ✔              |
+| [NPM (including Yarn, Pnpm)](docs/detectors/npm.md)                              | ✔                                               | ✔              |
+| [NuGet (including Paket)](docs/detectors/nuget.md)                               | ✔                                               | ✔              |
+| [Pip (Python)](docs/detectors/pip.md)                                            | ✔                                               | ✔              |
+| [Poetry (Python, lockfiles only)](docs/detectors/poetry.md)                      | ✔                                               | ❌              |
+| Ruby                                                                             | ✔                                               | ✔              |
+| Rust                                                                             | ✔                                               | ✔              |
 
 For a complete feature overview refer to [feature-overview.md](docs/feature-overview.md)
 
-# My favorite language/ecosystem isn't supported!
+## How To Use
 
-Component Detection is built with extensibility in mind! Please see our [CONTRIBUTING.md](CONTRIBUTING.md) to get started where you can find additional docs on adding your own detector.
+To clone and run this application, you'll need [Git](https://git-scm.com) and [.NET 6](https://aka.ms/download-dotnet) installed on your computer. From your command line:
 
+```bash
+# Clone this repository
+$ git clone https://github.com/microsoft/component-detection
 
-# Building and running Component Detection
-.NET Core 3.1 is currently in use, you can install it from https://dotnet.microsoft.com/download/dotnet/3.1
+# Go into the repository
+$ cd component-detection 
 
-The below commands mirror what we do to setup our CI environments:
+# Run the app
+$ dotnet run 
+```
+
+View the [detector arguments](docs/detector-arguments.md) for more information on how to use the tool.
 
-From the base folder:
-``` dotnet build ```
+## Download
 
-## Running in Visual Studio (2019+)
-1. open [ComponentDetection.sln](ComponentDetection.sln) in Visual Studio
-1. Set the Loader project as the startup project (rightclick-> Set as Startup Project)
-1. Set Run arguments for the Loader project (rightclick->properties->Debug)  
-	*Minimum:* `scan --SourceDirectory <Repo to scan>`
-1. Now, any time you make a change, you can press `F5`. This will build the changes, and start the process in debug mode (hitting any breakpoints you set)
+You can [download](https://github.com/microsoft/component-detection/releases/tag/latest) the latest version of Component Detection for Windows, macOS and Linux.
 
 ## Using Codespaces
```

Comment:
This feels like it should be a separate 'contributing' section. The needs of someone who wants to download CD are different from someone who wants to contribute back to it.